### PR TITLE
Rans push to quay

### PR DESCRIPTION
**Pull request by solashRan**
+ tag the image with our quay account as well
+ expose the full version of the image to allow scripts that build that build this image know it

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,10 +3,12 @@ DEB_PATH = "iguazio_deb"
 BINARY_NAME = "igz-fuse"
 RELEASE_VERSION = "0.6.1"
 DOCKER_HUB_USER = "iguaziodocker"
+QUAY_IO_USER = "iguazio"
+FULL_VERSION = $(IGUAZIO_VERSION:igz_%=%)-$(RELEASE_VERSION)
 
 .PHONY: build
 build: lint vet
-	docker build --tag $(DOCKER_HUB_USER)/flex-fuse:unstable .
+	docker build --tag $(DOCKER_HUB_USER)/flex-fuse:unstable --tag $(QUAY_IO_USER)/flex-fuse:unstable .
```
Comment (bivas) [inline Makefile]: No need to local tag the quay user

File: Makefile [hunk 2]
```diff
@@ -17,7 +19,9 @@ download:
 
 .PHONY: release
 release: check-req download build
-	docker tag $(DOCKER_HUB_USER)/flex-fuse:unstable $(DOCKER_HUB_USER)/flex-fuse:$(IGUAZIO_VERSION:igz_%=%)-$(RELEASE_VERSION)
+	docker tag $(DOCKER_HUB_USER)/flex-fuse:unstable $(DOCKER_HUB_USER)/flex-fuse:$(FULL_VERSION)
+	docker tag $(QUAY_IO_USER)/flex-fuse:unstable $(QUAY_IO_USER)/flex-fuse:$(FULL_VERSION)
```
Comment (bivas) [inline Makefile]: Just re-tag the docker version with quay name

File: Makefile [hunk 3]
```diff
@@ -3,10 +3,12 @@ DEB_PATH = "iguazio_deb"
 BINARY_NAME = "igz-fuse"
 RELEASE_VERSION = "0.6.1"
 DOCKER_HUB_USER = "iguaziodocker"
+QUAY_IO_USER = "iguazio"
```
Comment (bivas) [inline Makefile]: Isn't this user should be `quay.io/iguazio`?

**Review by bivas — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (solashRan) [reply Makefile]: no need for the ?  mark

**Comment by solashRan** [comment]
fixed comments

File: Makefile [hunk 4]
```diff
@@ -17,7 +18,9 @@ download:
 
 .PHONY: release
 release: check-req download build
-	docker tag $(DOCKER_HUB_USER)/flex-fuse:unstable $(DOCKER_HUB_USER)/flex-fuse:$(IGUAZIO_VERSION:igz_%=%)-$(RELEASE_VERSION)
+	docker tag $(DOCKER_HUB_USER)/flex-fuse:unstable $(DOCKER_HUB_USER)/flex-fuse:$(FULL_VERSION)
+	docker tag $(DOCKER_HUB_USER)/flex-fuse:unstable quay.io/iguazio/flex-fuse:$(FULL_VERSION)
```
Comment (bivas) [inline Makefile]: Don't inline. You should place it as VAR (as before).

**Review by bivas — CHANGES_REQUESTED** [review]

File: Makefile [hunk 5]
```diff
@@ -17,7 +19,9 @@ download:
 
 .PHONY: release
 release: check-req download build
-	docker tag $(DOCKER_HUB_USER)/flex-fuse:unstable $(DOCKER_HUB_USER)/flex-fuse:$(IGUAZIO_VERSION:igz_%=%)-$(RELEASE_VERSION)
+	docker tag $(DOCKER_HUB_USER)/flex-fuse:unstable $(DOCKER_HUB_USER)/flex-fuse:$(FULL_VERSION)
+	docker tag $(DOCKER_HUB_USER)/flex-fuse:unstable $(QUAY_IO_USER)/flex-fuse:$(FULL_VERSION)
+	echo $(FULL_VERSION) > VERSION
```
Comment (bivas) [inline Makefile:24]: This will do nothing.

**Review by bivas — CHANGES_REQUESTED** [review]

**Pull request merged**